tracer/src/Datadog.Trace/AppSec/Waf/Initialization/WafConfigurator.cs

```diff
@@ -48,7 +48,8 @@ public bool CheckVersionCompatibility()
             }
 
             // tracer >= 2.34.0 needs waf >= 1.11 cause it passes a ddwafobject for diagnostics instead of a ruleset info struct which causes unpredictable unmanaged crashes
-            if (tracerVersion is { Minor: >= 34, Major: >= 2 } && wafMajor == 1 && wafMinor <= 10)
+            if ((tracerVersion is { Minor: >= 34, Major: >= 2 } && wafMajor == 1 && wafMinor <= 10) ||
+                (tracerVersion is { Minor: >= 35, Major: >= 2 } && wafMajor == 1 && wafMinor < 13))
```

Comment:
Depending on whether you merge this before or after code freeze, you might need to bump the version condition?
```suggestion
                (tracerVersion is { Minor: >= 36, Major: >= 2 } && wafMajor == 1 && wafMinor < 13))
```

Reply:
thanks a lot for raising this, I surely would have missed it and 💥